src/main/java/com/hit/OneOrZeroKnapsackAlgoImpl.java

```diff
@@ -1,17 +1,17 @@
-package main.java.com.hit;
+package com.hit;
 
 import java.util.ArrayList;
 import java.util.Collection;
 
-public class OneOrZeroKnapsackAlgoImpl implements IAlgoKnapsack {
+public class OneOrZeroKnapsackAlgoImpl implements main.java.com.hit.IAlgoKnapsack {
         // the items which are put
         // in a knapsack of capacity W
         @Override
         public Collection<Integer> createShoppingCart(int[] v, int[] w, int totalWeight , int n)
         {
             int i, wi;
             int K[][] = new int[n + 1][totalWeight + 1];
-            ArrayList<Integer> collectedObjects = new ArrayList<>();
+            ArrayList<Integer> collectedObjects = new ArrayList<Integer>();
```

Comment:
Why you change this?
  ArrayList<Integer> collectedObjects = new ArrayList<Integer>()
I got a warning about it so I change this to -
 ArrayList<Integer> collectedObjects = new ArrayList<>();